File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int> q;
        for(int i: nums) q.push(i);
        int n = 1;
        while(n < k){
            q.pop();
            n += 1;
        }
        return q.top();
    }
};

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
import java.util.PriorityQueue;
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<Integer>(Collections.reverseOrder());
        for(int i: nums) pq.add(i);
        int n = 1;
        while (n < k){
            n += 1;
            pq.remove();
        }
        return pq.peek();
    }
}